staging/src/k8s.io/client-go/kubernetes/fake/clientset_generated.go

```diff
@@ -155,6 +155,33 @@ func NewSimpleClientset(objects ...runtime.Object) *Clientset {
 	return cs
 }
 
+// NewSimpleClientsetWithObjectTrackerOption returns a clientset that will respond with the provided objects.
+// its function is consistent with NewSimpleClientset, the difference is that NewSimpleClientsetWithObjectTrackerOption
+// will customize clientSet.ObecjectTracker, the current optional is custom watcher
```

Comment:
Just a minor type here. `clientSet.ObecjectTracker` to `clientSet.ObjectTracker`.

Reply:
Ok, thank you, I have edited it